examples/console/resources/console/console.deployment.yaml

```
@@ -20,9 +20,10 @@ spec:
       containers:
       ### API
       - name: console-api
-        image: quay.io/eyefloaters/console-api:0.0.31
+        image: registry.redhat.io/amq-streams/console-api-rhel8:TBD
```

```suggestion
        image: registry.redhat.io/amq-streams/console-api-rhel8:2.7
```
use x.y floating tag
https://errata.devel.redhat.com/package/show/amqstreams-console-api-container